digital_land/datatype/wkt.py

```diff
@@ -209,24 +214,27 @@ def normalise(self, value, default="", issues=None):
             # fixed/normalised geometry. To reduce precision,
             # round trip the geometry through shapely with 6 dp precision.
 
-            _wkt = dump_wkt(geometry, precision=6)
+            _wkt, issue = dump_wkt(geometry, precision=6)
+            if issue:
+                issues.log("geometry is too precise - fixed", "")
+
             geometry = shapely.wkt.loads(_wkt)
             validity = False
             i = 0
             while i < 3 and not validity:
                 geometry, issue = normalise_geometry(geometry)
                 if issue:
-                    issues.log("invalid geometry", issue)
+                    issues.log("invalid geometry provided - fixed", issue)
```

Comment:
This is where stuff gets a bit wild. we may need to rethink how we record these issues as it's a loop. specifically in this case we are primarily fixing errors introduced by attempting to reduce the precision, at this stage we have fix the issue caused by reducing precision but we will dump it out again below. I'm not against describing that we fixed this but I'm worried it could be misleading when examining the issue log